src/icon-tab-bar.tsx

```diff
@@ -42,23 +43,23 @@ export const IconTabBar: FC<Props> = ({ tabs, selectedTab, onTabSelected }) => {
         )}
         role="tab"
         aria-selected={isSelected}
-        aria-label={PROJECTS[project].label()}
+        aria-label={PROJECTS[project].label(msg)}
```

Comment:
Reading these as "label using msg" - would it be clearer as `msg(PROJECTS[project].label)` to avoid passing the `msg` helper into every label?

Reply:
Oh I see, because we're still using lit localize at build time to extract strings, we still need a `msg` call site to allow it to know which strings need translations. Is that it?

Reply:
Right, we're still using `lit/localize-tools`, which needs the argument to `msg` to be a string literal so it can extract it.